Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Functionality - Microsoft Store Limited to Private Store Only.json

```diff
@@ -8,6 +8,6 @@
       }
     }
   ],
-  "description": "Limit Microsoft Store To private store only (per https://support.microsoft.com/en-us/help/4339074/removing-uninstalling-reinstalling-microsoft-store-app-not-supported, the store should not be un-installed altogether, so we're instead limiting it to the private store only)",
+  "description": "What is it: Limits Microsoft Store to a private store only. \n Simeon Notes: Simeon has found that very few apps in the Microsoft store are worth installing. This policy allows you to restrict your users to only apps that you deem necessary. \n Learn More: https://docs.microsoft.com/en-us/microsoft-store/distribute-apps-from-your-private-store",
```

Comment:
**What does this do?**
Limits the Microsoft Store to applications of your choice (those in your private store). This policy allows you to restrict your users to only apps that you deem necessary.  

**Why should you use this?**
If you want to limit the applications which users may download to those in your private store.

**What is the end-user impact?**
Users may only use apps that you deem necessary. 

**Learn more**
[Distribute apps using your private store](https://docs.microsoft.com/en-us/microsoft-store/distribute-apps-from-your-private-store)